department-of-veterans-affairs/va.gov-cms
# VACMS-2538,2988: fix lists in wysiwyg & rename wysiwyg paragraph type to "Rich text"

## Description

See:

- #2538 
- #2988 

## Testing done

Updated spec tool & ran behat tests.

## Screenshots


## QA steps

1. Add a new 'CMS Help Page'
1. Click 'Add Content Block -> Rich Text'
1. Verify that ordered & unordered lists have consistent indentation/styling, and that bullets can be seen for both
1. Then validate Acceptance Criteria from issues
- [ ] The 'Rich text' paragraph type is present
- [ ] Other help text has been correctly updated
- [ ] List styling is correct

## Definition of Done
